tools/eventbrite.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Eventbrite
+questions:
+  - event-management
+---
+
+TTS uses [Eventbrite](https://www.eventbrite.com/) to manage registrations for events we host.
+
+## Rules
+
+The following are required for accessibility reasons:
+
+- Include the following in the [event description](https://www.eventbrite.com/support/articles/en_US/Multi_Group_How_To/how-to-create-an-event?lg=en_US#2-2):
+
+  > Thank you for your interest in this event. Please contact us directly at [email] if you need accessible accommodations to be able to attend or assistance with registering for the event.
```

Comment:
Still [getting consensus around what this language should be](https://docs.google.com/document/d/1gXLD66RmsUT27-uPkasKFv8i9Q1e3JAvAQTncNkEhas/edit?disco=AAAAKXvDq5A). I suggest we don't wait to merge this page — I'll send a follow-up pull request once finalized.